### ISSUE TITLE
FewRel fine-tuning?

### ISSUE THREAD

**Issue by demongolem-biz** [plkmo/BERT-Relation-Extraction#37]
@plkmo In main_task.py, the comments suggest you can fine-tune on either SemEval or FewRel.  However, the README file gives results that are not trained on an FewRel data.  It appears to me that only inference is taking place in regards to FewRel.  Is this so, or is there a way to fine-tune on my own FewRel data (and save a resultant model files)?

A few more words in the FewRel Task README.md file would also be helpful.

**Comment by plkmo** [comment]
Yes, FewRel fine-tuning have not yet been implemented, I have put as a to-do task
